firmware/src/peripherals/esp32.rs

```diff
@@ -0,0 +1,60 @@
+use super::Peripherals;
+use crate::aliases::ඞ::DelayConcrete;
+use crate::aliases::ඞ::I2cConcrete;
+
+use fugit::RateExtU32;
+
+use esp32_hal::{
+	clock::{ClockControl, CpuClock},
+	pac,
+	prelude::*,
+	timer::TimerGroup,
+	Rtc,
+};
+
+pub fn get_peripherals() -> Peripherals<I2cConcrete<'static>, DelayConcrete> {
+	let p = pac::Peripherals::take().unwrap();
+
+	let mut system = p.DPORT.split();
+	// The ESP-Wifi module requires 160MHz for cpu clock speeed
+	let clocks =
+		ClockControl::configure(system.clock_control, CpuClock::Clock240MHz).freeze();
```

Comment:
did you mean 240 or 160?

Reply:
I copy pasted from the esp32c3 code haha

Reply:
change comment to say 240MHz as that is what the esp32 example uses